datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonJob.scala

```diff
@@ -78,14 +68,18 @@ object DatasetComparisonJob {
     if (expectedExceptActual.count + actualExceptExpected.count == 0) {
       scribe.info("Expected and actual data sets are the same.")
     } else {
-      cmd.keys match {
+      cliOptions.keys match {
         case Some(keys) =>
-          handleKeyBasedErrorsPresent(keys, cmd.outPath, expectedExceptActual, actualExceptExpected)
+          handleKeyBasedErrorsPresent(keys.toSeq, cliOptions.outPath, expectedExceptActual, actualExceptExpected)
         case None =>
-          expectedExceptActual.write.format("parquet").save(s"${cmd.outPath}/expected_minus_actual")
-          actualExceptExpected.write.format("parquet").save(s"${cmd.outPath}/actual_minus_expected")
+          expectedExceptActual.write.format("parquet").save(s"${cliOptions.outPath}/expected_minus_actual")
+          actualExceptExpected.write.format("parquet").save(s"${cliOptions.outPath}/actual_minus_expected")
       }
-      throw DatasetsDifferException(cmd.refPath, cmd.newPath, cmd.outPath, expectedDf.count(), actualDf.count())
+      throw DatasetsDifferException(cliOptions.referenceOptions.path,
+                                    cliOptions.newOptions.path,
+                                    cliOptions.outPath,
+                                    expectedDf.count(),
+                                    actualDf.count())
     }
```

Comment:
nit: too much horizontal space. in multiline methods the every argument should be along on the line.